laborator/content/toolchain-decompilare/demo/goto/goto.c

```diff
@@ -0,0 +1,23 @@
+/*
+ * Author: Gabriel Mocanu <gabi.mocanu98@gmail.com>
+ */
```

Comment:
Same as above

Reply:
These are not general demos, they are more particular to me. I put this signature so I can identify them more easily.